File: app/src/main/java/com/example/androidcodelabapp/model/GithubUsersResponse.java
package com.example.androidcodelabapp.model;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

public class GithubUsersResponse {
    @SerializedName("items")
    ArrayList<GithubUsers> githubUsers;

    public GithubUsersResponse(){
        githubUsers = new ArrayList<>();

    }
    public ArrayList<GithubUsers> getGithubUsers(){
        return githubUsers;
    }
}

File: app/src/main/java/com/example/androidcodelabapp/model/GithubUsersResponse.kt
package com.example.androidcodelabapp.model

import com.google.gson.annotations.SerializedName

import java.util.ArrayList

class GithubUsersResponse {
    @SerializedName("items")
    private var githubUsers: ArrayList<GithubUser> = ArrayList()

    fun getGithubUsers(): ArrayList<GithubUser> = githubUsers
}
